doc/schemas/listclosedchannels.schema.json

```diff
@@ -9,8 +9,10 @@
   "properties": {
     "closedchannels": {
       "type": "array",
+      "added": "v23.05",
```

Comment:
These too...

Reply:
I suspected as much. I'll remove them and update the PR.

Reply:
I removed `"added": "v23.05",` from all the schema json files (where I introduced them).
If you want me to squash my part of this PR then let me know.